### PR TITLE
Update \s match characters

### PR DESCRIPTION
**Pull request by btea**

### CODE REVIEW

File: files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ Character classes distinguish kinds of characters such as, for example, distingu
           feed, line feed, and other Unicode spaces. Equivalent to
           <code
             >[
-            \f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]</code
+            \f\n\r\t\v\u0020\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]</code
           >. For example, <code>/\s\w*/</code> matches " bar" in "foo bar".
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md:165]:
```html
<code>[\f\n\r\t\v\u0020\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]</code>
```

File: files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ Character classes distinguish kinds of characters such as, for example, distingu
           Matches a single character other than white space. Equivalent to
           <code
             >[^
-            \f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]</code
+            \f\n\r\t\v\u0020\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]</code
           >. For example, <code>/\S\w*/</code> matches "foo" in "foo bar".
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md:177]:
```html
<code>[^\f\n\r\t\v\u0020\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]</code>
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Thanks for the PR. Next time, please fill in the PR description, because the motivation is clear to me but may not be to other reviewers who are less specialized with JS and regexes. This helps us review the PR more efficiently.

The current text is actually correct, but it's very confusing, because the space is implicitly there through the line break between `[` and `\f`, which turns into a space when rendered. However, due to the table width, the text wraps at that space. You can do some weird tricks by scaling down the page and changing `max-width`:

image

So to properly fix it, you need to also reformat the HTML in order to make the text make sense.

File: files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md (hunk 1 above)
Comment (btea) [reply files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md:165]: > ```
> <code>[\f\n\r\t\v\u0020\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]</code>
> ```

After I modify it like this, the rendered content will wrap after `-`, is this performance correct?
image

Comment (Josh-Cena) [reply files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md:165]: Yes, this is fine.

Comment (btea) [reply files/en-us/web/javascript/guide/regular_expressions/character_classes/index.md]: Updated.

**Review by Josh-Cena — APPROVED** [review]
Thanks, and congratulations on your first contribution here—welcome aboard!

**Pull request merged**